snippets/csharp/System.Linq/Queryable/AggregateTSource/queryable.cs

```diff
@@ -1339,19 +1354,21 @@ static void LastOrDefaultEx1()
         static void LastOrDefaultEx2()
         {
             // <Snippet46>
-            double[] numbers = { 49.6, 52.3, 51.0, 49.4, 50.2, 48.3 };
+            double[] numbers = [49.6, 52.3, 51.0, 49.4, 50.2, 48.3];
 
             // Get the last number in the array that rounds to 50.0,
             // or else the default value for type double (0.0).
             double last50 =
-                numbers.AsQueryable().LastOrDefault(n => Math.Round(n) == 50.0);
+                numbers
+                    .LastOrDefault(n => Math.Round(n) == 50.0);
 
             Console.WriteLine("The last number that rounds to 50 is {0}.", last50);
 
             // Get the last number in the array that rounds to 40.0,
             // or else the default value for type double (0.0).
             double last40 =
-                numbers.AsQueryable().LastOrDefault(n => Math.Round(n) == 40.0);
+                numbers
+                    .LastOrDefault(n => Math.Round(n) == 40.0);
```

Comment:
I'll stop commenting on it, but every sample in this file is to show a data source that uses `IQueryable`. The `AsQueryable()` calls should stay.

Reply:
not a fan as adds unnecessary overhead, and imho is bad guidance
but I will revert as-was to stay friends.

Yes I also prefer the lightweight record rather than class, so I will magic that in too.